### PR TITLE
receipt_root -> receipts_root

### PR DESCRIPTION
**Pull request by djrtwo**
Minor renaming for upcoming bellatrix spec release.

This was discussed and generally agreed upon on a public call. Due to the need to make a release patching the semantics of executePayload and forkchoiceUpdated, we are getting this in there as well

### CODE REVIEW

**Review by mkalinin — APPROVED** [review]
LGTM!

File: specs/bellatrix/beacon-chain.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ class ExecutionPayload(Container):
     parent_hash: Hash32
     fee_recipient: ExecutionAddress  # 'beneficiary' in the yellow paper
     state_root: Bytes32
-    receipt_root: Bytes32  # 'receipts root' in the yellow paper
+    receipts_root: Bytes32  # 'receipts root' in the yellow paper
```
Comment (hwwhww) [inline specs/bellatrix/beacon-chain.md:172]: [not strongly] It seems now we can consider removing the comment here.
```suggestion
    receipts_root: Bytes32
```

**Review by hwwhww — APPROVED** [review]
LGTM

**Review by lokera666 — COMMENTED** [review]
Best practice

**Pull request merged**